src/server/game/Spells/SpellEffects.cpp

```diff
@@ -534,14 +534,6 @@ void Spell::EffectSchoolDMG(SpellEffIndex effIndex)
             }
             case SPELLFAMILY_DRUID:
             {
-                // Ferocious Bite
-                if (m_caster->GetTypeId() == TYPEID_PLAYER && m_spellInfo->SpellFamilyFlags[3] & 0x1000)
```

Comment:
Belongs in different PR

Reply:
Is it? the idea was this code depends on the fact we're not handling the power_burn, and this PR fixed it so the code isn't needed anymore.
no?

Reply:
To me this belongs to ferocious bite PR